docs/asset/cfd_trading.puml

```diff
@@ -0,0 +1,50 @@
+@startuml
+
+participant TakerDaemon
+participant MakerDaemon
+participant MakerAutomation
+participant BitMex
+
+
+== Offer Protocol ==
+TakerDaemon->MakerDaemon: Connect
+|||
+loop heartbeat
+|||
+    MakerDaemon->BitMex: Check bid & ask prices
```

Comment:
Not 100% correct: we subscribe to the `quoteBin1m` topic, meaning, we get unconditionally a new quote every minute.